src/website/views/getting-started/development-workflow.pug

```diff
@@ -145,9 +145,9 @@ p.-text Method decorator should be multi-line
     ...
   }
 
-p.-text Use private variables and methods as much as possible: They are useful to detect dead code and enforce encapsulation. Note that this is a feature which TypeScript provides to help hardening your code, but using private, public or protected does not make a difference in the actual JavaScript output.
+p.-text Use private variables and methods as much as possible; they are useful to detect dead code and enforce encapsulation. Note that this is a feature which TypeScript provides to help hardening your code, but using private, public or protected does not make a difference in the actual JavaScript output.
 
-p.-text Code with Method/Prop/Event/Component decorators should have jsdocs: This allows documentation generation and for a better user experience in an editor that has TypeScript intellisense. <em>Stencil auto-generated documentation will be included in the Chi documentation pages. </em>
+p.-text Code with Method/Prop/Event/Component decorators should have #[jsdocs]; this allows documentation generation and for a better user experience in an editor that has TypeScript intellisense. <em>Stencil auto-generated documentation will be included in the Chi documentation pages. </em>
```

Comment:
Pug is rendering as an html tag and the text jsdocs is no longer visible on the page